plugins/discovery-ec2/build.gradle

```diff
@@ -6,7 +6,7 @@ esplugin {
 
 dependencies {
   compile 'com.amazonaws:aws-java-sdk-ec2:1.10.19'
-  compile 'org.apache.httpcomponents:httpclient'
+  compile "org.apache.httpcomponents:httpclient:${versions.httpclient}"
```

Comment:
Warning on this one (and on s3). 
It might pull a wrong version.

AWS SDK defines it's own version